dtos/deviceprofile_test.go

```diff
@@ -37,7 +37,7 @@ var testDeviceProfile = models.DeviceProfile{
 		Properties: models.ResourceProperties{
 			ValueType: common.ValueTypeInt16,
 			ReadWrite: common.ReadWrite_RW,
-			Others:    testTags,
+			Optional:  testTags,
```

Comment:
Why using `testTags`? Optional is not for tags, so please use something more relevant to what we expect it to be used for.